TTHAnalysis/python/analyzers/MT2Skimmer.py

```diff
@@ -45,20 +50,24 @@ def process(self, event):
         zll_mt2 = event.mt2_zll
         zll_met_pt = event.zll_met_pt
 
-        #QCD skim to be added in OR next time
-        #skimmingSelection="isGolden && ((nJet30>1 && nlep==0 && met_pt > 30. &&  (diffMetMht < 0.5*met_pt) && mt2>50. ))"
 
+        signalSkim = ( (ht > 200 and nJet30 >= 1 and ( (nJet30>=2 and mt2>200.) or nJet30==1 ) ) 
+                          and ((ht<1000. and met_pt>200.) or (ht>1000 and  met_pt>30)) )
+
+        gammaSkim = (gamma_ht > 200 and gamma_nJet30 >= 1 
+                     and ((gamma_nJet30>=2 and gamma_mt2>200.) or (gamma_nJet30==1 and gamma_ht>200.)) 
+                     and ((gamma_ht<1000. and gamma_met_pt>200.)or(gamma_ht>1000 and  gamma_met_pt>30)) ) 
+
+        zllSkim = (zll_ht > 200. and nJet30 >= 1 and ((nJet30==1 and zll_ht>200.) or (nJet30>1  and zll_mt2>200.)) 
+                   and ((zll_ht<1000. and zll_met_pt>200.) or (zll_ht>1000 and  zll_met_pt>30)) )
+
+        qcdSkim = ( (nJet30>1 and nLep==0 and met_pt > 30. and  (diffMetMht < 0.5*met_pt) and mt2>50. ) 
+                    or 
+                    (nJet30==2 and met_pt>200. and ht>200. and nLep==0 and diffMetMht < 0.5*met_pt and deltaPhiMin>0.3) 
```

Comment:
@mschoene has performed some checks that this new sign is indeed ok - i'll let her comment